sample_data_config/opensearch/bigquery-to-opensearch.yaml

```diff
@@ -74,33 +81,145 @@ bigQueryToOpenSearch:
                   type: "text"
                 publication_date:
                   type: "date"
-                s2_specter_embedding_v1:
-                  type: "knn_vector"
-                  dimension: 768
-                  method:
-                    # Note: we need Lucene for filter support
-                    name: "hnsw"
-                    space_type: "l2"
-                    engine: "lucene"
-                    parameters:
-                      # Default values: https://opensearch.org/docs/latest/search-plugins/knn/knn-index/
-                      ef_construction: 512
-                      m: 16
+                s2:
+                  properties:
+                    author_list:
+                      type: "nested"
+                    specter_embedding_v1:
+                      type: "knn_vector"
+                      dimension: 768
+                      method:
+                        # Note: we need Lucene for filter support
+                        name: "hnsw"
+                        space_type: "l2"
+                        engine: "lucene"
+                        parameters:
+                          # Default values: https://opensearch.org/docs/latest/search-plugins/knn/knn-index/
+                          ef_construction: 512
+                          m: 16
+                europepmc:
```

Comment:
It would probably be more clear to configure the index settings outside the pipeline. We could also update the mappings again when we come to EuropePMC but that doesn't seem ideal either.